index.html

```diff
@@ -0,0 +1,268 @@
+            <img src="images/project-1.png" class="project_photos">
+            <div class="project_content_description">
+              <h2>Creative Agency</h2>
+              <h3>Full Branding, Website, App</h3>
+            </div>
+          </div>
+          <div class="project_container_content_right pages">
+            <img src="images/project-2.png" class="project_photos">
+            <div class="project_content_description">
+              <h2>Digital Marketing</h2>
+              <h3>Logo, Website & App</h3>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- ######################################### TOGETHER SECTION ######################################### -->
+    <div class="together_container">
+      <div class="together_background">
+        <img src="images/bubble-12.png" class="together_photo bubble12">
+        <img src="images/bubble-13.png" class="together_photo bubble13">
+        <img src="images/bubble-14.png" class="together_photo bubble14">
+        <img src="images/bubble-16.png" class="together_photo bubble16">
+        <img src="images/chat-group.png" class="together_photo chat">
+        <img src="images/bill-pay.png" class="together_photo bill">
+        <img src="images/laptop-poses.png" class="together_photo laptop">
+      </div>
+      <div class="together_title">
+        <h7>Let's Work Together</h7>
+        <h1>Need a successful project?</h1>
+      </div>
+      <div class="together_project">Estimate Project</div>
+    </div>
+    <!-- ######################################### FOOTER SECTION ######################################### -->
+    <div class="footer_container">
+      <div class="social_bar">
+        <img src="images/facebook.png">
+        <img src="images/twitter.png">
+        <img src="images/pinterest.png">
+        <img src="images/linkedin.png">
+        <img src="images/instagram.png">
+        <img src="images/yt.png">
+      </div>
+      <div class="main_footer_section">
+        <div class="left_section">
+          <div class="left_section_container">
+            <h1>Get in touch!</h1>
+            <h3>many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected</h3>
+          </div>
+          <div class="subscribe">
+            <div class="emial_inbox">
+              <img src="images/email.png" class="email">
+              Email address
+            </div>
+            <div class="subscribe_button">Subscribe</div>
+          </div>
+        </div>
+        <div class="right_section">
+          <div class="services">
+            <h4>Services</h4>
+            <h5>Logo & Branding</h5>
+            <h5>Website Development</h5>
+            <h5>Mobile App Development</h5>
+            <h5>Search Engine Optimalization</h5>
+            <h5>Pay-Per-Click</h5>
+            <h5>Social Media Marketing</h5>
+          </div>
```

Comment:
Znowu wszystko na nagłówkach. Generalnie ważne jest żeby używać znaczników, które są do tego przeznaczone. Oczywiście wszystko możemy zrobić na divach, ale tutaj po prostu wychodzić nieznajomość znaczników.

Wygląda to jak lista, więc powinna być lista. Poza tym widać, że w to się powinno dać kliknąć.

```suggestion
          <div class="services">
            <h4>Services</h4>
            <ul class="services_list">
              <li class="services_item">
                <a href="/">Logo & Branding</a>
              </li>
            </ul>
            ...
          </div>
```